### DISCUSSION TITLE
A question about IPX algorithm

### DISCUSSION THREAD

**Discussion by tusiqi1** [ERGO-Code/HiGHS#1551] [Q&A]
I'm not sure if I'm right.

I found that the IPX algorithm does not use a central path, and the self-dual nonlinear system of equations. Why solve for KKT conditions directly?  Can the addition of these two techniques accelerate LP solving?

References:
Implementation of an interior pointmethod with basis preconditioning
The Mosek Interior Point Optimizer for Linear Programming: An Implementation of the Homogeneous Algorithm

**Comment by jajhall** [comment]
IPX does follow the central path
